spec/rails_helper.rb

```diff
@@ -4,11 +4,24 @@
 abort("The Rails environment is running in production mode!") if Rails.env.production?
 require 'spec_helper'
 require 'rspec/rails'
+require 'webmock/rspec'
 
 Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
 
 ActiveRecord::Migration.maintain_test_schema!
 
+# Enforce that tests can't make unmocked http requests
+WebMock.disable_net_connect!(allow_localhost: true)
+WebMock.enable!
+
+VCR.configure do |config|
```

Comment:
Should we do an environment variable thing like we do in our other tests to decide whether or not we fetch new things?

Reply:
Yeah I can add that. Right now a new cassette will only be created if a test declares one that doesn't exist. Adding that flag and setting it to false would result in a failure in the test instead of creation. I don't think that would come up very often.